client/blocks/reader-post-card/style.scss

```diff
@@ -400,6 +406,13 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	}
 }
 
+// 3 line excerpt for thumbnail cards
+.reader-post-card.card.has-thumbnail:not(.is-gallery) {
```

Comment:
@samouri I structured it this way because of a difficultly targeting text-only cards for the 2 line treatment over 960px (there's no display type for them as far as I can see).